packages/sales-api-service/src/services/permissions.service.js

```diff
@@ -86,8 +88,9 @@ const getAgeCategory = (birthDate, issueDate) => {
   const dob = moment(birthDate)
   const issue = moment(issueDate)
   const diff = issue.diff(dob, 'years', true)
+  const seventeenthBirthday = dob.clone().add(ADULT_AGE, 'years')
 
-  if (isJunior(diff)) {
+  if (issue.isBefore(seventeenthBirthday)) {
     return 'J'
   } else if (isSenior(diff)) {
```

Comment:
Just wondering, if using the diff like this was an issue for junior licences, is it also an issue for seniors?

I think it'd be worth having unit tests for birthdays which are in this rounding-up error zone.